numpyro/distributions/continuous.py

```diff
@@ -326,6 +329,39 @@ def variance(self):
         return np.where(self.concentration <= 2, np.inf, a)
 
 
+@copy_docs_from(Distribution)
+class Gumbel(Distribution):
+    arg_constraints = {'loc': constraints.real, 'scale': constraints.positive}
+    support = constraints.real
+    reparametrized_params = ['loc', 'scale']
+
+    def __init__(self, loc=0., scale=1., validate_args=None):
+        self.loc, self.scale = promote_shapes(loc, scale)
+        batch_shape = lax.broadcast_shapes(np.shape(loc), np.shape(scale))
+
+        super(Gumbel, self).__init__(batch_shape=batch_shape,
+                                     validate_args=validate_args)
+
+    def sample(self, key, sample_shape=()):
+        standard_gumbel_sample = random.gumbel(key, shape=sample_shape + self.batch_shape + self.event_shape)
+        return self.loc + self.scale * standard_gumbel_sample
+
+    @validate_sample
+    def log_prob(self, value):
+        z = (value - self.loc) / self.scale
+        return (np.exp(-z) - z) - np.log(self.scale)
```

Comment:
I think you miss `-` sign before exp term.

Reply:
Thank you, I did indeed.